sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/CloudEventRequestContent.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Core.Serialization;
+
+namespace Azure.Messaging.EventGrid
+{
+    internal class CloudEventRequestContent : RequestContent
```

Comment:
Would be nice to have some tests specifically for this type.

Reply:
I just updated an existing test so that we now have full coverage for this type.

Reply:
Does it cover retries?

Reply:
Working on that as we speak 😄